tests/raftstore/cluster.rs

```diff
@@ -163,10 +167,11 @@ impl<T: Simulator> Cluster<T> {
     }
 
     pub fn call_command(&self,
+                        node_id: u64,
```

Comment:
Not really. TiDB caches 'storeID -> serverAddr' relationships in a cache, which maybe out of date.